### ISSUE TITLE
Redefine map length to exclude empty intro time

### ISSUE THREAD

**Issue by IceDynamix** [Quaver/Quaver#1190]
TL;DR: Including empty intros into the displayed length does not have any value, may interfere with, for example, the NPS number and may cause confusion under the players.

Example: 
[Wir fliegen (Overdrive difficulty)](https://quavergame.com/mapsets/map/1022) is a pretty short map with an intro of 8.3 seconds, the map ends at 37.2 seconds giving a gameplay length of around 29 seconds. The current length for the map is shown as 37 seconds though, because it includes the empty intro time. 

Wir fliegen is a short, but very dense chordjack map. There aren't any breaks, the NPS number therefore helps predicting the difficulty when the rating system fails. It's currently shown as 22 NPS, by taking the 827 objects and dividing them by 37 seconds (`827/37=22.35`). But that's far off from the *actual* NPS, which is around 29. We can confirm that by applying the calculation again, but this time with the 29 seconds of gameplay length instead of the displayed 37 seconds (`827/29=28.52`). We can give further confirmation by comparing the data with a selfmade NPS graph I made, which shows following:
![image](https://user-images.githubusercontent.com/22303902/67642750-39bca200-f90f-11e9-8d93-72f719537e1e.png)
